runtime/vm/JFRChunkWriter.hpp

```diff
@@ -24,6 +24,7 @@
 
 #include "j9cfg.h"
 #include "j9.h"
+#include "omrformatconsts.h"
```

Comment:
Again, this is not appropriate.

Reply:
https://github.com/eclipse-openj9/openj9/pull/20158#discussion_r1874049537

Reply:
When used with `snprintf()`, yes, but not with `omrstr_printf()` or other OMR API: this should be using OMR functions exclusively for such things.